dali/treeview/CMakeLists.txt

```diff
@@ -44,6 +44,9 @@ if (WIN32)
         )
 
     add_definitions("-D_AFXDLL")
+    set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS} /wd4996" )
+	set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4996" )
```

Comment:
Is this another case where we should open a ticket to address the warning - 'deprecated method in use' ?

Reply:
We could, but as I far as I know, this is used by ConfigEnv only and ConfigEnv is going away so this may not be on our priority.